helm/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -438,18 +475,39 @@ spec:
         resources:
           {{- toYaml . | trim | nindent 10 }}
         {{- end }}
-      {{ if and (.Values.cleanupKubeProxy) (not (eq .Values.kubeProxyReplacement "disabled")) }}
-      - name: cleanup-kube-proxy-iptables
+      {{- if and .Values.waitForKubeProxy (ne $kubeProxyReplacement "strict") }}
```

Comment:
This was a custimuzation we made to support upgrading from was CNI to cilium in vintage. We still need it

Reply:
👍  I'll add it back!

Reply:
This now correct?

Reply:
I am not very lucid right now, but I'd say no, not correct.
What I need is this container here
https://github.com/giantswarm/cilium-app/blob/main/helm/cilium/templates/cilium-agent/daemonset.yaml#L441